### ISSUE TITLE
Web demo: linting takes 120 ms

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#721]
> john_hillington_: lint() takes 128 ms and markEditorText 4ms
> john_hillington_: i7 8700k + Chrome 101
> john_hillington_: Windows

**Comment by strager** [comment]
Offending text:
```js
const oh iuccupation = lagyugyngughhjgyugage + " coder";
co uih uihui huiuhuigyuhihiunht langhuihui uage = "JavaScript";
occuphhuihuihuiation = "Senior " + occupation;

console.log(`Welcome, ${ocupation}!`);
```

**Comment by strager** [comment]
Fixed in commit 7bfeb3150b1764bd84206691b8f7cdec4e6b3856 and deployed. 😳
